src/server/ua_server_discovery.c

```diff
@@ -5,6 +5,8 @@
 #include "ua_server_internal.h"
 #include "ua_client.h"
 #include "ua_config_standard.h"
+#include "ua_types_generated.h"
+#include "ua_types_generated_handling.h"
```

Comment:
Same here... Porbably not needed, because the .c file did not change, so it should not require new includes?